### PR TITLE
create-user

### PR DESCRIPTION
**Pull request by ghost**
## Description
Create User : layout create user , validation , save users;
## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- explain select * from users\G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: users
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 32
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)

### CODE REVIEW

File: app/Http/Controllers/Admin/UsersController.php [hunk 1]
```diff
@@ -16,4 +17,34 @@ public function index()
...
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $this->validate(request(), [
```
Comment (at-thinhnguyen) [inline app/Http/Controllers/Admin/UsersController.php]: validate ở Request luôn nha

File: app/Http/Controllers/Admin/UsersController.php [hunk 2]
```diff
@@ -16,4 +17,34 @@ public function index()
...
+            'name'=>'bail|required',
+            'email'=>'bail|required|unique:users',
+            'password'=>'bail|required|min:9',
+        ]);
+        User::create([
```
Comment (at-thinhnguyen) [inline app/Http/Controllers/Admin/UsersController.php]: ko cần trim đâu, chổ email nên validate thêm `email` nữa

File: resources/views/admin/user/create.blade.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                <div class="form-group">
+                  <label for="exampleInputName">Name</label>
+                  <input type="text" class="form-control" id="exampleInputName" name="name" placeholder="Enter name">
+                </div>
+                <div class="form-group">
+                  <label for="exampleInputEmail">Email</label>
+                  <input type="email" class="form-control" id="exampleInputEmail" name="email" placeholder="Enter name">
+                </div>
+                <div class="form-group">
+                  <label for="exampleInputPassword">Password</label>
```
Comment (at-thinhnguyen) [inline resources/views/admin/user/create.blade.php]: bỏ máy chử example đê, thêm trường nhập confirm email nữa

**Review by ghost — CHANGES_REQUESTED** [review]
kiểm tra comment của a nha

**Review by ghost — APPROVED** [review]

**Pull request merged**